pyproject.toml

```diff
@@ -1,12 +1,64 @@
 [build-system]
 
 requires = [
-    "setuptools>=42",
+    "setuptools",
+    "setuptools-scm",
     "wheel"
 ]
-
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "odxtools"
+description = "Utilities to work with the ODX standard for automotive diagnostics"
+license = {text = "MIT"}
+authors = [
+    {name = "Katrin Bauer", email = "katrin.bauer@mbition.io"},
+    {name = "Andreas Lauser", email = "andreas.lauser@mbition.io"},
+    {name = "Ayoub Kaanich", email = "kayoub5@live.com"},
+]
+maintainers = [
+    {name = "Andreas Lauser", email = "andreas.lauser@mbition.io"},
+    {name = "Ayoub Kaanich", email = "kayoub5@live.com"},
+]
+readme = "README.md"
+requires-python = ">=3.8"
+keywords = ['can', 'can bus', 'DoIP', 'odx', 'pdx', 'obd', 'uds', 'automotive', 'diagnostics']
+classifiers = [
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.8',
+    'Development Status :: 5 - Production/Stable',
+    'Environment :: Console',
+    'Intended Audience :: Developers',
+    'License :: OSI Approved :: MIT License',
+    'Operating System :: POSIX',
+]
+dependencies = [
+    "bitstruct >= 6.0.0",
+    "argparse_addons",
+    "PyInquirer",
+    "jinja2",
+    "python-can",
+    "can-isotp",
```

Comment:
`can` is needed by the snoop tool, but besides this there currently there is nothing. (the `snoop` tool needs to implement its own ISO-TP state machine because it must work in "passive mode"...)

A much more problematic dependency is `PyInquirer` (because it seems like it constantly like to break installations). Since this is only needed by the somewhat obscure `browse` tool, I put it into the `browse-tool` dependency group which is not selected by default...